src/shadows.js

```diff
@@ -0,0 +1,40 @@
+import * as Blockly from 'blockly/core';
```

Comment:
Just realized I haven't been checking this super closely, but this file probably needs a license?